src/ExchangeContainer/ExchangeContainer.js

```diff
@@ -1,17 +1,65 @@
 import React, { useState, useEffect } from 'react'
 import { Route } from 'react-router-dom'
-import { Link, NavLink } from 'react-router-dom';
-import './ExchangeContainer.scss';
+import { Link, NavLink } from 'react-router-dom'
```

Comment:
I'll need to do some clean up to remove unused imports, etc. I plan to do that on a clean up branch (or as I work if I remember!)